classes/class-wc-connect-nux.php

```diff
@@ -1,5 +1,16 @@
 <?php
 
+if ( ! class_exists( 'Jetpack_Install_Status' ) ) {
+	abstract class Jetpack_Install_Status {
```

Comment:
So that we don't have to instantiate it, as described in this [PHP doc](http://php.net/manual/en/language.oop5.abstract.php). Since this class doesn't rely on object properties - they're all basically static properties - there's no need for an instance. Does this make sense in the general realm of PHP & WP?